.travis.yml

```diff
@@ -1,8 +1,11 @@
 ---
+sudo: required
+services:
+    - docker
 language: python
 python: 2.7
 cache: pip
-install:
-  - "pip install vim-vint==0.3.9"
-script:
-  - "vint -s ."
+install: |
+    pip install vim-vint==0.3.9
+script: |
+    vint -s . && make test
```

Comment:
Setting two `env` items would be one way to do it. Set a variable different in different environments for Travis, and run one command or the other based on the variable.